src/Screens/Cart/index.tsx

```diff
@@ -3,89 +3,45 @@ import React from "react";
 import Icon from 'react-native-vector-icons/FontAwesome'
 import styles from "./styles";
-import PrimaryBtn from '../../components/UI/Button/primaryBtn'
-import Colors from '../../components/UI/Reusable'
+import Header from "../../components/header";
+import CartCard from '../../components/cartCard'
+import Buttonn from "../../components/button/button";
+import { images } from "../../assets/arrays/theme";
 
 const Cart = ({navigation,route}) =>{
     const item = route.params
 
     return (
     <View style={styles.container}>
-           <View style={styles.header}>
-                <Icon name="arrow-left" size={25} color={Colors.darkGrey}
-                onPress={()=>navigation.navigate('details')}/>
-                
-                <Icon name="user-circle-o" size={28} color={Colors.darkGrey}
-                onPress={()=>navigation.navigate('details')}/>
-           </View>
+        
+           <Header icon1={"arrow-left"} icon2={"user-circle-o"} size={28}
+                onpress1 ={()=>{ navigation.navigate('details') }}
+                onpress2 ={()=>{ navigation.navigate('details') }}
+                image= {false} />
 
            <View style={styles.titleView}>
                 <Text style={styles.title}>My Bag</Text>
                 <Text style={styles.subTitle}>Check and Pay your Item</Text>
            </View>
 
-            <View style={styles.cartCard}> 
-    
-                   <Image source={item.item.item.img} style={styles.cardImg}/>
-
-                    <View style={styles.cardText}>
-
-                        <Text style={{fontSize:25,fontWeight:'bold'}}>{item.item.item.name}</Text>
-                        <Text style={{fontSize:17,paddingVertical:5}}>{item.item.item.info}</Text>
-                        <Icon name="close" size={20} style={styles.cardIcon}/>
-                        <View style={styles.Price}>
-                            <Icon name="rupee" size={20}/>
-                            <Text style={{fontWeight:'bold',fontSize:17,marginLeft:5}}>{item.item.item.price}</Text>
-                        </View>
-
-                        <View style={styles.quant}>
-                            <Icon name="minus" size={20} style={styles.quantIcon}/>
-                            <Text style={styles.quantText}>1</Text>
-                            <Icon name="plus" size={20} style={styles.quantIcon}/>
-                            <Text style={styles.quantText}>|</Text>
-                            <Text style={styles.quantText}>43</Text>
-                            <Icon name="chevron-down" size={20} style={styles.quantIcon}/>
-                        </View>
-
-                        
-                    </View>
-            </View>
-            
-
-            <View style={styles.cartCard}> 
-    
-                   <Image source={require('../../assets/woman/w6.jpg')} style={styles.cardImg}/>
-
-                    <View style={styles.cardText}>
+           <CartCard  image={item.item.item.img}
```

Comment:
If all values are going through the same object. You can use the spread operator to shorten passed props.
eg.

`<CartCard {...item} />`